18.0 daniel

danyyy10 opened the pull request:
Refactor company section visibility handling and improve template attributes for invoice checkbox,

danyyy10 committed:
Add maintainer name to README.md

danyyy10 committed:
Fix formatting in README.md by removing redundant line

danyyy10 committed:
Add VAT validation and company data fields for Romanian users

- Implement SCSS styles for company data section and VAT input validation.
- Enhance portal templates to include optional company data fields with tooltips and validation for VAT number.
- Modify website sale templates to add company data fields and validation during checkout, including responsive design adjustments.
- Introduce loading animations and feedback messages for user interactions.

danyyy10 committed:
Refactor company section visibility handling and improve template attributes for invoice checkbox

danyyy10 committed:
Enhance ESLint configuration for VAT validation scripts and set priority for portal templates

danyyy10 committed:
[FIX] deltatech_website_vat_validation: Remove trailing whitespace for pre-commit checks

danyyy10 committed:
[FIX] deltatech_website_vat_validation: Set priority=99 for xpath replace to pass pre-commit

danyyy10 committed:
[FIX] deltatech_website_vat_validation: Remove unused function parameters (eslint errors)

danyyy10 committed:
[FIX] deltatech_website_vat_validation: Fix remaining ESLint errors and reorder XML attributes

danyyy10 committed:
[FIX] deltatech_website_vat_validation: Replace dangerous position=replace with position=inside for icons

danyyy10 commented:
dsgrgrd

dhongu commented:
O parte din functii deja exista in https://github.com/OCA/l10n-romania/tree/18.0/l10n_ro_partner_create_by_vat

dhongu reviewed (changes requested):
Toate textele sunt in Romana si trebuie sa fie in Engleza

danyyy10 commented:
Păi am extins și la nivel de vizibilitate în website..ai testat? acesta nu
era .

On Sun, 7 Dec 2025 at 19:54, Dorin Hongu ***@***.***> wrote:

> *dhongu* left a comment (dhongu/deltatech#2239)
> <https://github.com/dhongu/deltatech/pull/2239#issuecomment-3622683699>
>
> O parte din functii deja exista in
> https://github.com/OCA/l10n-romania/tree/18.0/l10n_ro_partner_create_by_vat
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dhongu/deltatech/pull/2239#issuecomment-3622683699>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZGZSHC7NAFBCZFH67KUJDD4ARSXDAVCNFSM6AAAAACOGGZCFGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMRSGY4DGNRZHE>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

danyyy10 commented:
Teste la nivel de website?..cele vizibile sau la nivel de cod ?

On Sun, 7 Dec 2025 at 19:57, Daniel Popa ***@***.***> wrote:

> Păi am extins și la nivel de vizibilitate în website..ai testat? acesta nu
> era .
>
> On Sun, 7 Dec 2025 at 19:54, Dorin Hongu ***@***.***> wrote:
>
>> *dhongu* left a comment (dhongu/deltatech#2239)
>> <https://github.com/dhongu/deltatech/pull/2239#issuecomment-3622683699>
>>
>> O parte din functii deja exista in
>> https://github.com/OCA/l10n-romania/tree/18.0/l10n_ro_partner_create_by_vat
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/dhongu/deltatech/pull/2239#issuecomment-3622683699>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AZGZSHC7NAFBCZFH67KUJDD4ARSXDAVCNFSM6AAAAACOGGZCFGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMRSGY4DGNRZHE>
>> .
>> You are receiving this because you modified the open/close state.Message
>> ID: ***@***.***>
>>
>

danyyy10 committed:
feat: Update VAT validation for Romanian companies

- Refactor JavaScript for frontend validation of Romanian VAT (CUI) in website checkout, including ANAF auto-fill functionality.
- Enhance user experience with improved validation messages and tooltips.
- Update SCSS styles for consistency with Bootstrap standards and improve visual feedback for validation states.
- Modify portal and website templates to support new validation logic and improve user guidance for entering company details.
- Translate comments and messages to English for better accessibility.

danyyy10 committed:
[FIX] website_vat_validation: Simplify Romania country code check and improve phone input validation

danyyy10 committed:
[FIX] deltatech_website_vat_validation: Correct Romanian translations and fix special character issues

danyyy10 committed:
[FIX] deltatech_website_vat_validation: Update Romanian translations for VAT validation messages and improve clarity

danyyy10 committed:
[FIX] pyproject.toml: Add missing dependencies for VAT validation project

danyyy10 committed:
[FIX] __manifest__.py: Update license from OPL-1 to AGPL-3

danyyy10 committed:
[IMP] website_vat_validation: clarificare mesaj fallback ANAF

danyyy10 committed:
[FIX] website_vat_validation: alinieri prettier si fallback ANAF

danyyy10 committed:
CI: ruleaza pre-commit doar pe fișierele din PR

danyyy10 committed:
[IMP] templates: improve formatting for quantity restrictions and VAT number instructions

danyyy10 committed:
[IMP] vat_validation: enhance duplicate checks and user feedback in checkout process

dhongu reviewed (changes requested):
Modificari in mai multe module

danyyy10 committed:
[IMP] website_vat_validation: enhance duplicate checks and user feedback with ANAF integration

danyyy10 committed:
[IMP] vat_validation: improve error messages, enhance country selection logic, and update templates for better user guidance

dhongu commented on deltatech_sale_multiple_website/views/templates.xml:
```diff
@@ -3,11 +3,14 @@
         <xpath expr="//div[@id='product_option_block']" position="after">
             <div
-                t-if="product.qty_multiple and product.qty_multiple &gt; 1 or product.qty_minim and product.qty_minim &gt; 1"
+                t-if="hasattr(product, 'qty_multiple') and product.qty_multiple and product.qty_multiple &gt; 1 or hasattr(product, 'qty_minim') and product.qty_minim and product.qty_minim &gt; 1"
```
pentru ce ai pus astea ?
